### PR TITLE
Bug:54372 - Deprecate distributed mode in MLCP

### PR DESCRIPTION
**Pull request by yunzvanessa**

### CODE REVIEW

File: mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java [hunk 1]
```diff
@@ -152,10 +152,6 @@ public static int runCommand(String[] args) throws IOException {
                 MODE + ": " + mode);
             return 1;
         }
-        if (MODE_DISTRIBUTED.equalsIgnoreCase(mode)) {
-            LOG.warn("MLCP distributed mode will be disabled in the next " +
```
Comment (jmakeig) [inline mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java:156]: Can we proactively put a link to the docs for more information?

Comment (yunzvanessa) [reply mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java:156]: A link in the error message or a link in the documentation/release note?

Comment (jmakeig) [reply mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java:156]: Some sort of look-up in the log message, like an error code or a URL, so it’s to find the the doc (or more likely via Google).

Comment (yunzvanessa) [reply mlcp/src/main/java/com/marklogic/contentpump/ContentPump.java:156]: Okay, I will add a url to our mlcp user guide which has more details on the deprecation.

**Pull request merged**